### PR TITLE
Fix #98: Make hyperlinks open in new tab

### LINKED ISSUE #98: Make hyperlinks(HTML links) open in new tab
Currently all hyperlinks open in the same tab.

### PR DESCRIPTION
**Pull request by abarya**
- I made changes to the files **base.html** and **index.html** so that if we click any hyperlink on the 
  website, the browser would open it in a new tab.
- It fixes the issue #98.

### CODE REVIEW

File: oshc/main/templates/index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h3 class="panel-title">{{ session.title }}</h3>
 						<div class="line-gap-10px"></div>
 						<i>{{ session.start_date }} - {{ session.end_date }}</i>
 						<div class="line-gap-10px"></div>
-						<button type="button" class="btn btn-lg btn-primary sRegister">
+						<button type="button" class="btn btn-lg btn-primary s">
```
Comment (vaibhavsingh97) [inline oshc/main/templates/index.html]: extraneous change. @abhishekarya286 Please don't change anything that are not relevant to the issue.

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by vaibhavsingh97** [comment]
LGTM
Awesome!! @abhishekarya286 Congratulations on your first PR  :tada: :tada:. You have done a great job.

File: oshc/main/templates/index.html (hunk 1 above)
Comment (abarya) [reply oshc/main/templates/index.html]: Sorry, I did this by mistake. I have edited it now.

**Comment by abarya** [comment]
Thanks @vaibhavsingh97!!

File: oshc/main/templates/index.html (hunk 1 above)
Comment (vaibhavsingh97) [reply oshc/main/templates/index.html]: @abhishekarya286 It's ok to make mistakes :wink: 
One learn from his mistakes

**Review by vaibhavsingh97 — APPROVED** [review]

**Comment by vaibhavsingh97** [comment]
@abhishekarya286 You are so fast!! :+1:This PR is ready to merge.

File: oshc/main/templates/index.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	<div class="container">
 		<h3>Our Mission</h3>
 		<p class="bi">
-			<b>"</b>Open Source Help Community is an a virtual platform to help people to start contributing to open source projects<b>"</b>
+			<b>"</b>Open Source Help Community is a virtual platform to help people to start contributing to open source projects<b>"</b>
```
Comment (nikhita) [inline oshc/main/templates/index.html:18]: thanks for this fix too. :)

**Review by nikhita — APPROVED** [review]

**Comment by nikhita** [comment]
@abhishekarya286 Congratulations on getting your first PR merged in OSHC! :smile:

**Comment by abarya** [comment]
Thanks @nikhita

**Pull request merged**